utilities/backupable/backupable_db_test.cc

```diff
@@ -748,13 +749,13 @@ class BackupableDBTest : public testing::Test {
 
   void DeleteLogFiles() {
     std::vector<std::string> delete_logs;
-    db_chroot_env_->GetChildren(dbname_, &delete_logs);
+    db_chroot_env_->GetChildren(dbname_, &delete_logs).PermitUncheckedError();
```

Comment:
I would rather see either:

1. `if (db_chroot_env_->GetChildren(...).ok()) {`
2. return Status on error from the function and call ASSERT/EXPECT on the two calls to `DeleteLogFiles`